code/style.css

```diff
@@ -1,7 +1,183 @@
+
```

Comment:
In this file some wierd indention is going on. In some places it's two spaces and in some four spaces. It would be great if the file follows one of them!